tests/ci/codebuild/linux-x86/amazonlinux-2_gcc-7x_intel-sde.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 0.2
+
+phases:
+  pre_build:
+    commands:
+      - if [ $(gcc -dumpfullversion) == 7.3.1 ]; then echo "Found correct gcc version 7"; else gcc --version && echo "gcc version mismatch" && exit 1; fi
+      # # Based on Interl SDE README, SELinux should be turned off to allow pin to work.
+      # https://software.intel.com/content/www/us/en/develop/articles/intel-software-development-emulator.html#system-configuration
+      - if [ $(getenforce) == 'Disabled' ]; then echo "SELinux is disabled. Disabling SELinux is needed by sde to allow pin work." ; else echo "SELinux should be turned off to allow sde pin to work." && exit 1; fi
```

Comment:
Does it make sense to fail the build if SELinux is not disabled since it is a requirement for the SDE tests?

Reply:
The fail is in else block.

Reply:
Sorry, I completely missed the `exit 1`

Reply:
no worry. it's very long if else :)